### PR TITLE
feat(node_rebalance): document node rebalancing

### PR DESCRIPTION
**Pull request by savonarola**
Documentation to the rebalancing feature: https://github.com/emqx/emqx/pull/8597

### CODE REVIEW

File: en_US/advanced/rebalancing.md [hunk 1]
```diff
@@ -0,0 +1,364 @@
...
+```
+
+### Get Cluster-wide Status
+
+```
+curl -s -u admin:public -H "Content-Type: application/json" -X GET 'http://127.0.0.1:8081/api/v4/load_rebalance/global_status'
+{
+  "rebalances": {},
+  "evacuations": {
+    "emqx1@127.0.0.1": {
```
Comment (Rory-Z) [inline en_US/advanced/rebalancing.md]: I have a question, this key appears to be mutable, which is unfriendly to json parsing, maybe we can like this:
```
{
  "evacuations": [
    {
      "node": "emqx1@127.0.0.1"
    }
  ]
}
```
@savonarola @zmstone what do you think

Comment (Rory-Z) [reply en_US/advanced/rebalancing.md]: @savonarola @zmstone ping again

Comment (savonarola) [reply en_US/advanced/rebalancing.md]: ✅  I'll make a PR to expose friendlier API

Comment (savonarola) [reply en_US/advanced/rebalancing.md]: https://github.com/emqx/emqx-enterprise/pull/1479

File: en_US/advanced/rebalancing.md [hunk 2]
```diff
@@ -0,0 +1,365 @@
...
+        "current_connected": 0
+      },
+      "state": "waiting_takeover",
+      "session_recipients": [
+        "emqx3@127.0.0.1",
+        "emqx2@127.0.0.1"
+      ],
+      "session_goal": 0,
+      "session_eviction_rate": 5,
+      "connection_goal": 0,
```
Comment (Rory-Z) [inline en_US/advanced/rebalancing.md:358]: Hi, @savonarola I have a question: what's different between `connection_goal` and `initial_connected`, which one should I use to determine that evacuations have been completed ?

**Comment by wivwiv** [comment]
This section has been added to the documentation in another PR.
https://docs.emqx.com/en/enterprise/v4.4/advanced/rebalancing.html

**Pull request closed without merging**